platform/security/idp/security-idp-client/src/main/java/org/codice/ddf/security/idp/client/IdpMetadata.java

```diff
@@ -61,7 +70,13 @@
 
   public void setMetadata(String metadata) {
     this.metadata = metadata;
+    eraseEntityDescriptions();
+  }
+
+  private synchronized void eraseEntityDescriptions() {
     entryDescriptions.getAndSet(null);
```

Comment:
❓ Now that we are synchronized, do we really need to keep this attribute as an atomic one?

Reply:
I think originally it was @coyotesqrl who did this, I think he did atomic for performance reasons. But if everything is synchronized, no need anymore.

Reply:
That was my assumption as well.

Reply:
I really hate to mess with what Richard did. We had a lot of hard-to-fix timing issues and this class went through many changes trying to fix them.